geopolars/src/geoseries.rs

```diff
@@ -26,6 +26,11 @@ pub trait GeoSeries {
     /// geometry.
     fn envelope(&self) -> Result<Series>;
 
+    /// Returns a GeoSeries of geometries representing the convex hull of each geometry.
+    ///
+    /// The convex hull of a geometry is the smallest convex Polygon containing all the points in each geometr
+    fn convex_hull(&self) -> Result<Series>;
```

Comment:
Nit: I was thinking of keeping this in alphabetical order. Not sure if that's important or just ocd 🤷‍♂️

Reply:
Happy to move it to the correct alphabetic location :-)